### PR TITLE
FreeBSD dns plugin packages

### PR DESCRIPTION
**Pull request by nod0n**
#### Pull Request (PR) description
Add the FreeBSD packages for dns-route53 dns-rfc2136.

### CODE REVIEW

**Review by ekohl — APPROVED** [review]

**Comment by ekohl** [comment]
Test failures look relevant.

**Comment by nod0n** [comment]
> Test failures look relevant.

Yes. I'm trying to find what the problem is.

File: spec/classes/plugin/dns_rfc2136_spec.rb [hunk 1]
```diff
@@ -21,6 +21,8 @@ class { 'letsencrypt':
           'python3-certbot-dns-rfc2136'
         when 'RedHat-7', 'CentOS-7'
           'python2-certbot-dns-rfc2136'
+        when 'FreeBSD-12', 'FreeBSD-13'
+          'py38-certbot-dns-rfc2136'
```
Comment (smortex) [inline spec/classes/plugin/dns_rfc2136_spec.rb:25]: On line 48 bellow, config file path is hardcoded as `/etc/letsencrypt/dns-rfc2136.ini`, I would expect something under `/usr/local/etc` on FreeBSD :wink:

Comment (nod0n) [reply spec/classes/plugin/dns_rfc2136_spec.rb:25]: https://github.com/voxpupuli/puppet-letsencrypt/pull/268/commits/1448922831d3516ba1084cc8fae891d0a7bd4358 should do the trick.

**Review by smortex — APPROVED** [review]
Looks good

**Pull request merged**